extensions/resteasy-reactive/quarkus-resteasy-reactive/deployment/src/main/java/io/quarkus/resteasy/reactive/server/deployment/ResteasyReactiveProcessor.java

```diff
@@ -37,6 +37,7 @@
 import java.util.stream.Collectors;
 import java.util.stream.Stream;
 
+import jakarta.enterprise.inject.spi.DeploymentException;
```

Comment:
@geoand  We are looking at backports with @rsvoboda and I'm surprised we use this exception here?

Not a biggy and not blocking but it looks weird.

Reply:
I used this one because this one is guaranteed to work with hot reload (Arc uses it all over the place).
The previous one we were using was preventing hot reload.

Reply:
OK. At some point we might need to introduce an exception for all these deployment issues. I know I was annoyed by not having something that was generic enough for that in Quarkus itself.

Anyway, all good, thanks.